src/modules/navigator/rtl_direct.cpp

```diff
@@ -426,7 +422,7 @@ rtl_time_estimate_s RtlDirect::calc_rtl_time_estimate()
 
 	RTLState start_state_for_estimate;
 
-	if (isActive()) {
```

Comment:
isActive does not come from the mission, but from the navigator_mode, which rtl_direct inherits. However, it is not correctly set, since the navigator_mode run function is not triggered. Maybe it would be better to change the rtl.cpp function to call the run function instead of the on_active function directly.

Reply:
We wanted to not alter it anymore when it is activated. One use case when it is triggered due to low battery, you don't want to change to a location further away. You can still actively override it and change the location then. Also for situation when you change the home position to your position periodically when you move, you want to have the ability to step aside when it RTLs without it following anymore